lib/web/scripts/node-join/install.sh

```diff
@@ -817,6 +818,9 @@ install_from_file() {
         elif check_exists yum; then
             log "Found 'yum' package manager, using it"
             PACKAGE_MANAGER_COMMAND="yum -y localinstall"
+		elif check_exists zypper; then
+			log "Found 'zypper' package manager, using it"
+			PACKAGE_MANAGER_COMMAND="zypper --non-interactive install"
```

Comment:
nit: the diff here looks weird, check that we're using tabs vs spaces consistently